File: php/99.php
 <?php
for ($x = 99; $x > 0; $x--) {
    echo sprintf('%d bottles of beer on the wall, %d bottles of beer%s', $x, $x, PHP_EOL);
    echo sprintf('Take one down and pass it around, %d bottles of beer on the wall%s', $x-1, PHP_EOL);
}
?> 

File: fortran/99.f90
program beers
  integer :: i
  do i = 99, 1, -1
    print '(i2, " bottles of beer on the wall, ", i2, " bottles of beer")', i, i
    print '("Take one down and pass it around, ", i2, " bottles of beer on the wall")', i - 1
  end do
end program beers